src/PaymentSuite/PaymillBundle/Resources/views/Paymill/scripts.html.twig

```diff
@@ -7,58 +7,96 @@
     /**
      * Validation for paymill payment
      */
-    $("#paymill_form").submit(function(){
 
-        $("#paymill_view_submit").attr("disabled", "disabled");
+    var oPaymill = {
+        oForm : document.getElementById('paymill_form'),
+        oErrorContainer : document.getElementById('paymill_payment_error'),
+        oSubmit : document.getElementById('paymill_view_submit'),
+        nCardNumbers1 : 0000,
+        nCardNumbers2 : 0000,
+        nCardNumbers3 : 0000,
+        nCardNumbers4 : 0000,
+        nCardNumbers : 0000,
+        sCardHolder : '',
+        sExpiryMonth : '',
+        sExpiryYear : '',
+        nCvv : 000,
+        bOkNumber : false,
+        bOkExpiry : false,
+        bOkCvv : false
+    };
 
-        //join all cc numbers in one
-        var cc = $('#paymill_view_credit_card_1').val() + $('#paymill_view_credit_card_2').val() + $('#paymill_view_credit_card_3').val() + $('#paymill_view_credit_card_4').val();
-        var cardholder = $('#paymill_view_credit_card_owner').val();
-        var expiry_month = $('#paymill_view_credit_card_expiration_month').val();
-        var expiry_year = $('#paymill_view_credit_card_expiration_year').val();
-        var cvv = $('#paymill_view_credit_card_security').val();
 
-        var $paymill_payment_error = $('#paymill_payment_error');
-        $paymill_payment_error.html('');
+    oPaymill.oForm.setAttribute('onsubmit','return submitPaymill()');
```

Comment:
With the listener you need to create a fallback for IE8 browsers and below, with this method the code is more stable and compatile.